src/audio/google/google_rtc_audio_processing.c

```diff
@@ -602,18 +604,16 @@ static int google_rtc_audio_processing_prepare(struct processing_module *mod,
 		return -EINVAL;
 	}
 
+	struct comp_buffer *b0 = list_first_item(&dev->bsource_list, struct comp_buffer, sink_list);
+	struct comp_buffer *b1 = list_next_item(b0, sink_list);
+
+	cd->aec_reference_source = (is_ref_buffer(dev, b0) ? 0 : 1);
+	cd->raw_microphone_source = (cd->aec_reference_source == 1) ? 0 : 1;
```

Comment:
how about `1 - cd->aec_reference_source`

Reply:
Lets keep it simple, if we need to remap IDs then lets have a comment saying why/how.